content/docs/2.1/scalers/mongo.md

```diff
@@ -0,0 +1,80 @@
+# mongo with ScaledJob
+
+## Trigger Specification
+This specification describes the mongo trigger that scales based on result of Mongo query.
+
+The trigger always requires the following information:
```

Comment:
Can you align with the approach from https://github.com/kedacore/keda-docs/blob/master/content/docs/2.1/scalers/azure-service-bus.md#L31-L38 please?